uftrace_print.h

```diff
@@ -0,0 +1,101 @@
+#if defined(__clang__)
+#define UFT_DISABLE_OPT __attribute__((optnone))
+#elif defined(__GNUC__)
+#define UFT_DISABLE_OPT __attribute__((optimize(0)))
+#else
+#define UFT_DISABLE_OPT
+#endif
+
+
+#ifdef __cplusplus
+
+template <typename T>
+UFT_DISABLE_OPT
+void uftrace_print_cxx(const char *X, T)
+{
+}
+
+__attribute__((noinline))
+void uftrace_print_cxx(const char *X, const char* s)
+{
+    volatile const char c = *s;
+}
```

Comment:
Maybe you could define it as a static function to prevent multiple definition error.

Reply:
I just changed it to "static" function. Thanks.